### ISSUE TITLE
Accessibility: Return to cart link missing for zoomed in users

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43640]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

There are styles set that remove the return to cart link from the checkout page on mobile:
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-actions-block/style.scss#L28

This also removes the return to cart link for low-vision users who are zoomed in. In many cases, this may mean they cannot get back to the cart easily, especially if the mini cart is in use, which is hidden on the checkout page.

This is a [failure of WCAG "reflow" success criterion.](https://www.w3.org/WAI/WCAG21/Techniques/failures/F102)

Instead of hiding it, this should be visible and located after the payment button.

### Expected behavior

The return to cart link will be visible even when zoomed in or given the mobile view.

### Actual behavior

The return to cart link is hidden.

### Steps to reproduce

View the checkout page on mobile or with your browser zoomed in.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.
